app/assets/javascripts/admin/tests/admin/acceptance/admin-install-theme-modal-test.js

```diff
@@ -39,4 +39,13 @@ acceptance("Admin - Themes - Install modal", function (needs) {
       "private repo checkbox unchecked"
     );
   });
+
+  test("modal can be auto-opened with the right query params", async function (assert) {
+    await visit("/admin/customize/themes?repoUrl=testUrl&repoName=testName");
+    assert.ok(queryAll(".admin-install-theme-modal"), "modal is visible");
```

Comment:
`assert.ok(queryAll()) is a no-no. When `queryAll` uses jQuery under the hood, that assertion is always true, because `$()` returns a truthy object. Use `query` instead:

```suggestion
    assert.ok(query(".admin-install-theme-modal"), "modal is visible");
```

(needs an extra import)